packages/react/src/patterns/sections/CardSectionImages/README.md

```diff
@@ -0,0 +1,140 @@
+import { CardSectionImages } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+const CardSectionImages = [
+  {
+    image: {
+      defaultImage: 'https://picsum.photos/id/1003/1056/480',
+      alt: 'cards with image',
+    },
+    eyebrow: 'Topic',
+    heading: 'Natural language processing.',
+    cta: {
+      href: 'https://www.example.com',
+    },
+  },
+  {
+    image: {
+      defaultImage: 'https://picsum.photos/id/1018/1056/480',
+      alt: 'cards with image',
+    },
+    eyebrow: 'Blog',
+    heading: 'Natural language processing.',
+    cta: {
+      href: 'https://www.example.com',
+    },
+  },
+  {
+    image: {
+      defaultImage: 'https://picsum.photos/id/1076/1056/480',
+      alt: 'cards with image',
+    },
+    eyebrow: 'Topic',
+    heading: 'Natural language processing.',
+    cta: {
+      href: 'https://www.example.com',
+    },
+  },
+  {
+    image: {
+      defaultImage: 'https://picsum.photos/id/102/1056/480',
+      alt: 'cards with image',
+    },
+    eyebrow: 'Blog',
+    heading: 'Serving society ethically in the age of Artificial Intelligence.',
+    cta: {
+      href: 'https://www.example.com',
+    },
+  },
+  {
+    image: {
+      defaultImage: 'https://picsum.photos/id/1032/1056/480',
+      alt: 'cards with image',
+    },
+    eyebrow: 'Topic',
+    heading: 'Serving society ethically in the age of Artificial Intelligence.',
+    cta: {
+      href: 'https://www.example.com',
+    },
+  },
+];
+
+function App() {
+  return (
+    <CardSectionImages cards={CardSectionImages} heading="Read more about it" />
+  );
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+> 💡 Don't forget to import the card-section styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+## Props
+
+| Name      | Required | Data Type | Default Value | Description                                                                                                                                             |
+| --------- | -------- | --------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `heading` | YES      | String    | null          | Cards with or without images heading.                                                                                                                   |
+| `cards`   | YES      | Array     | null          | Array of card objects. See [Card](https://github.com/mkothur/ibm-dotcom-library/blob/master/packages/react/src/patterns/sub-patterns/Card/README.md) 👀 |
+| `theme`   | NO       | String    | 'white'       | Color theme for pattern. See `theme` below.                                                                                                             |
+
+## Card accepts the folloing information for this `CardSectionImages` pattern
```

Comment:
This is good, but I would give it a proper title, and add this as the copy. I'd also take it down a level in hierarchy.

```markdown
### Card props

Card accepts the folloing information for this `CardSectionImages` pattern

> 👀 See [Card](https://github.com/mkothur/ibm-dotcom-library/blob/master/packages/react/src/patterns/sub-patterns/Card/README.md) 

// table goes here
```

Reply:
ok, done